skyportal/tests/api/test_photometric_series.py

```diff
@@ -2125,3 +2128,278 @@ def test_get_series_paged(
     assert 1 <= len(data['data']['series']) <= 2
     page2_ids = [ps['id'] for ps in data['data']['series']]
     assert set(page1_ids).isdisjoint(page2_ids)
+
+
+def test_allowed_bandpasses():
+    # will remove this test once I add TESS to the bandpasses
+    from skyportal.enum_types import ALLOWED_BANDPASSES
+    from pprint import pprint
+
+    pprint(ALLOWED_BANDPASSES)
```

Comment:
What is this for exactly?

Reply:
Ahh yes, I forgot quite a few things in there while I was working... Good catches on all of these. 

I am thinking about leaving this in until we get the TESS filter merged so I can safely add the demo data with the correct filter...

Reply:
Let me make this even more obvious so it doesn't get left in there by mistake.